code/API_definitions/sim-swap-subscriptions.yaml

```
@@ -1040,6 +1040,12 @@ components:
                 status: 422
                 code: "MULTIEVENT_SUBSCRIPTION_NOT_SUPPORTED"
                 message: "Multi event types subscription not managed"
+            DeviceNotApplicable:
+              description: Service is not available for the provided device
+              value:
+                status: 422
+                code: DEVICE_NOT_APPLICABLE
+                message: The service is not available for the provided device.

```

To align with Sim Swap

```suggestion
            PhoneNumberNotApplicable:
              description: Service is not available for the provided phone number
              value:
                status: 422
                code: PHONE_NUMBER_NOT_APPLICABLE
                message: The service is not available for the provided phone number.

```